README.md

```diff
@@ -1,36 +1,36 @@
 
-PY_PG_TOOLS:
+# Py_pg_tools
 
     A powerful tool to use PostgreSQL in an easier and faster way. It allows
     the possibility to:
 
-    - Change the owner of a database and all of its tables.
-    - Make a custom backup of a PostgreSQL database or a whole cluster.
-    - Delete a PostgreSQL database.
-    - Get some information about some PostgreSQL databases, users or its
+    * Change the owner of a database and all of its tables.
+    * Make a custom backup of a PostgreSQL database or a whole cluster.
+    * Delete a PostgreSQL database.
+    * Get some information about some PostgreSQL databases, users or its
       current active connections.
-    - Clone a PostgreSQL database.
-    - Restore a database or cluster's backup in PostgreSQL.
-    - Add to, delete or get some instructions from the user's crontab file, to
+    * Clone a PostgreSQL database.
+    * Restore a database or cluster's backup in PostgreSQL.
+    * Add to, delete or get some instructions from the user's crontab file, to
       execute automatically the program in a specific date and time.
-    - Terminate some connections to PostgreSQL.
-    - Delete a group of backups depending on a very customizable options.
-    - Vacuum some PostgreSQL databases.
+    * Terminate some connections to PostgreSQL.
+    * Delete a group of backups depending on a very customizable options.
+    * Vacuum some PostgreSQL databases.
 
     See more information about py_pg_tools and its modules' characteristics
     letters specifying the argument -h or --help in console. For example
     (supposing you are in the same directory of py_pg_tools.py):
 
-    - python3 py_pg_tools.py -h
+    * python3 py_pg_tools.py -h
 
     If you want to see more information about a specific module, write its
     characteristic letter and then the argument -h or --help. For example,
     in case you need information about how to make a backup (supposing you are
     in the same directory of py_pg_tools.py):
 
-    - python3 py_pg_tools.py B -h
+    * python3 py_pg_tools.py B -h
 
-IMPORTED PACKAGES:
+# Imported packages:
```

Comment:
This should probably be second level title (also the rest): `## Imported packages:`